packages/pwa-kit-runtime/package.json

```diff
@@ -37,7 +37,7 @@
     "cross-env": "^5.2.1",
     "express": "^4.18.2",
     "header-case": "1.0.1",
-    "http-proxy-middleware": "0.21.0",
+    "http-proxy-middleware": "^2.0.6",
```

Comment:
I updated this dependency for the purposes of this spike. Upgrading required minimal changes and did not seem to have broken anything.